### ISSUE TITLE
Insert new TreeNode how to make it show 

### ISSUE THREAD

**Issue by erikswed** [bmelnychuk/AndroidTreeView#113]
I´m adding 3 TreeNodes like

-N1
----N2 
------N3

In the N2 node I´m creating a new node named N33 and doing `Node.addChild(node);`
and I expected my tree to look like this but nothing happens!
-N1
----N2 
------N3
------N33
   
How do I update invalidate the tree? I try `androidTreeView.getView().invalidate();` but that gives
"IllegalStateException: The specified child already has a parent. You must call removeView() on the child's parent first."

**Comment by MFV96** [comment]
i  have a question.please help me. ' container' .what is..?

**Comment by erikswed** [comment]
Check out wiki
https://en.wikipedia.org/wiki/Container

**Comment by mohitmadaan27** [comment]
@erikswed  have you found any solution?

**Comment by erikswed** [comment]
Yes in my case I just do `androidTreeView.expandAll();` from within N2. In my case the N2 is responsible for adding the nod.
